include/RED4ext/Api/v1/GameState.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+namespace RED4ext
+{
+struct CGameApplication;
+
+namespace v1
+{
+/**
+ * @brief Result type for game state hooks.
+ */
+enum class EGameStateResult : uint8_t
+{
+    Running,  /**< Hook has not yet finished. Until the state changes, hook continues to operate. */
```

Comment:
I think this comment is missing the most important part: "Running" prevents game state transition.